### DISCUSSION TITLE
Carla Crashes again and again when i launch make launch command

### DISCUSSION THREAD

**Discussion by kamraneng** [carla-simulator/carla#8384] [General]
I have tried carla first time in my PC. My PC have following specifications:
intel i5 13th generation and 16 GB Ram Nvidia Geforce RTX 4060

I followed all steps but on when i tried make launch, ue editor loaded upto 75% and then crashed. Now this is happening again and again. Please tell me what might be the probem? Thank you.

**Comment by NB-sys** [comment]
检查一下RAM是否太小
